user/articles/170-content-publication-management/03-change-lists/01-intro.markdown

```diff
@@ -1,9 +1,26 @@
----
-header-id: change-lists
----
-
 # Change Lists
 
-[TOC levels=1-4]
+Change Lists provide a way to track assets and how they're displayed system
+wide. You can group changes into working *sets* and publish them when they're
+ready. This prevents users from seeing changes that aren't ready for production;
+you can manage a change list and provide edits to your site in the background
+and push to the live environment when they're ready for your audience.
+
+A change list is not tied to a specific site. This means that you can track
+multi-site environments with a single change list.
+
+Change lists support two major changes:
+
+- Asset addition, edit, or deletion
+- Asset display (e.g., page, layouts, widget preferences)
+
+| **Note:** Change lists support for pages and layouts are still in development
+| and not available for @product@ 7.2 GA1.
+
+The following content types are supported by change lists at this time:
+
+- Web Content
+- DDM Template
```

Comment:
I would use Web Content Template and Structure here as DDM is more like a framework and many other assets like DL and DDL and Forms use it, but currently we limited the scope to Journal on purpose.